### PR TITLE
Revert "Revert "Doc updates for compression  on continuous aggregate feature""

### PR DESCRIPTION
**Pull request by Loquacity**
Reverts timescale/docs#730

### CODE REVIEW

File: timescaledb/how-to-guides/continuous-aggregates/compression-on-continuous-aggregates.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ 
+
+## Compression policies on continuous aggregates
+Before  setting up a compression policy on a continuous aggregate, you should
+set up a refresh policy. The compression policy interval should be set so that
+actively refreshed regions are not compressed. This is to prevent refresh
+policies from failing. For example, consider a refresh policy like this:
+
+```sql
+SELECT add_continuous_aggregate_policy('cagg_name', refresh_start=>'30 days', refresh_end=>'1 day', '1 h');
```
Comment (ryanbooz) [inline timescaledb/how-to-guides/continuous-aggregates/compression-on-continuous-aggregates.md:46]:
```suggestion
SELECT add_continuous_aggregate_policy('cagg_name', start_offset=>'30 days', end_offset=>'1 day', '1 h');
```

File: api/add_compression_policy.md [hunk 2]
```diff
@@ -2,18 +2,21 @@
...
 automatically in the background after it reaches a given age.
 
-Note that compression policies can only be created on hypertables that already
-have compression enabled, e.g., via the [`ALTER TABLE`][compression_alter-table] command
-to set `timescaledb.compress` and other configuration parameters.
+Note that compression policies can only be created on hypertables or continuous
+aggregates that already have compression enabled. Use the [`ALTER TABLE`][compression_alter-table] command
+to set `timescaledb.compress` and other configuration parameters for hypertables.
+Use [`ALTER MATERIALIZED VIEW`][compression_continuous-aggregate] command to
+enable compression on continuous aggregated
```
Comment (NunoFilipeSantos) [inline api/add_compression_policy.md:9]:
```suggestion
enable compression on continuous aggregates
```

**Review by NunoFilipeSantos — APPROVED** [review]
Fixed a small typo.

**Review by jacobprall — APPROVED** [review]

**Comment by Loquacity** [comment]
@NunoFilipeSantos Ping when you're ready for this to merge.

**Pull request merged**